src/components/menu-top/LoadShow.vue

```diff
@@ -0,0 +1,130 @@
+<template>
+  <div class="modal-card">
+    <header class="modal-card-head">
+      <p class="modal-card-title">
+        Load Show
+      </p>
+    </header>
+
+    <section class="modal-card-body">
+      <b-field class="file">
+        <b-upload
+          v-model="file"
+          accept=".shw"
+          @input="loadShow"
+        >
+          <a class="button is-primary">
+            <b-icon icon="upload" />
+            <span>Click to load</span>
+          </a>
+        </b-upload>
+        <span
+          v-if="file"
+          class="file-name"
+        >
+          {{ file.name }}
+        </span>
+      </b-field>
+      <span
+        v-if="parseError"
+      >
+        <b-message
+          title="Error"
+          type="is-danger"
+          aria-close-label="Close message"
+        >
+          {{ parseError }}
+        </b-message>
+      </span>
+      <span
```

Comment:
We don't want this `span` to be rendered in the html, so let's use `template` (it won't be rendered). See `Grapher.vue` for an example https://github.com/calband/calchart-redesign/blob/master/src/components/grapher/Grapher.vue#L21
```suggestion
      <template
```

Reply:
Going to address by adding the v-if directly in the fields.